bundles/org.openhab.binding.matter/src/main/java/org/openhab/binding/matter/internal/util/MatterUIDUtils.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.matter.internal.util;
+
+import java.math.BigInteger;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.matter.internal.MatterBindingConstants;
+import org.openhab.core.thing.ThingTypeUID;
+
+/**
+ * Utility class for creating and manipulating the UIDs for Matter thing types.
+ * 
+ * @author Dan Cunningham - Initial contribution
+ */
+@NonNullByDefault
+public class MatterUIDUtils {
+
+    /**
+     * Node thing types will have a UUID like matter:node_1234567890
+     *
+     * @param nodeId the node ID of the Node
+     * @return a ThingTypeUID for a Node
+     */
+    public static ThingTypeUID nodeThingTypeUID(BigInteger nodeId) {
+        return new ThingTypeUID(MatterBindingConstants.BINDING_ID,
+                MatterBindingConstants.THING_TYPE_NODE.getId() + "_" + nodeId);
```

Comment:
I know this comment is potentially very annoying at this point in time, but I think this should be:
```suggestion
                MatterBindingConstants.THING_TYPE_NODE.getId() + "-" + nodeId);
```

according to our [naming convention](https://www.openhab.org/docs/developer/guidelines.html#naming-convention)?